src/Connection/StreamConnection.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+declare(ticks=1);
```

Comment:
Ticks have huge overhead and generally are not a good idea. May I suggest to check Async signals implemented in PHP 7.2?
https://wiki.php.net/rfc/async_signals